src/object/brick.cpp

```diff
@@ -62,6 +62,17 @@ Brick::hit(Player& player)
   try_break(&player);
 }
 
+void
+Brick::update(float dt_sec)
+{
+  for (auto& explosion : Sector::get().get_objects_by_type<Explosion>())
+  {
+    if (get_bbox().grown(8).contains(explosion.get_bbox()))
+      try_break(nullptr);
+  }
+  Block::update(dt_sec);
+}
+
```

Comment:
This code belongs in the explosion, this should not be handled by bricks. Possible solutions would be to a) retain the current explosion size but diminish its lifetime or b) give the hurting sequence a smaller radius than normal.